buildSrc/src/main/kotlin/com/toasttab/protokt/shared/ProtoktExtension.kt

```diff
@@ -23,7 +23,8 @@ const val MANIFEST_VERSION_PROPERTY = "Implementation-Version"
 open class ProtoktExtension {
     var protocVersion = DEFAULT_PROTOBUF_VERSION
     var toolsVersion: String? = null
-    var publishProto: Boolean = false
+    var publishProto = false
+    var respectJavaPackage = true
```

Comment:
any reason to have this configurable vs always fall back to java_package? (like scalapb, for example)

Reply:
My thoughts were migration - if a project is using protobuf-java and wants to transition to protokt, always falling back on java_package means any code used from libraries (to be replaced with a protokt-generated library) can't coexist. Once migrated, they can re-enable the option to get the proper package names back.

Reply:
sounds good